src/components/FileUploadForm.vue

```diff
@@ -180,9 +181,10 @@ export default Vue.extend({
       this.uploading = true;
 
       try {
-        await api.uploadTable(workspace, fileName, {
+        await api.uploadNetwork(workspace, fileName, {
           type: selectedType.queryCall as UploadType,
           data: file,
+          url: host,
```

Comment:
If my [comment](https://github.com/multinet-app/multinetjs/pull/44#pullrequestreview-728659778) on the corresponding multinetjs PR is correct, then this will need to change.